tests/test_quantum_info_clifford.py

```diff
@@ -56,7 +56,7 @@ def test_clifford_from_circuit(backend, measurement):
         c.add(gates.M(*np.random.choice(3, size=2, replace=False)))
 
     result = clifford_backend.execute_circuit(c)
-    obj = Clifford.from_circuit(c, engine=backend)
+    obj = Clifford.from_circuit(c, engine=get_engine(backend))
```

Comment:
Not urgent, but you could also parametrize on something else than `backend` (i.e. put an `engine` argument, or `clifford_engine`, already in the test's parameters)